README.md

```diff
@@ -319,8 +319,10 @@ couchbackup.backup(
 ### Restore
 
 The `restore` function takes a readable stream containing the data emitted
-by the `backup` function. It uploads that to a Cloudant database, which
-should be a **new** database.
+by the `backup` function and uploads that to a Cloudant database.
+
+_Note:_ A target database should be a **new and empty** database to avoid
+possible documents' update conflicts.
```

Comment:
I think this a _must_ (particularly if we're doing #376) and I think as a requirement of the restore then the _why_ isn't adding a lot of value.
```suggestion
_Note:_ A target database must be a **new and empty** database.
```